src/client/game.cpp

```diff
@@ -3694,10 +3694,10 @@ void Game::handleDigging(const PointedThing &pointed, const v3s16 &nodepos,
 	camera->setDigging(0);  // Dig animation
 }
 
-
 void Game::updateFrame(ProfilerGraph *graph, RunStats *stats, f32 dtime,
 		const CameraOrientation &cam)
 {
+	static int resetHWBuffers = 0;
```

Comment:
should probably be `thread_local` (even if there's only one thread who does this)

Reply:
Should it be? Even if there were multiple thread doing the rendering, what matter is the amount the rendering work done in total, not per thread per work, right? So while it does not make a difference here, it seems logically should be global.

Thinking about it now... It should be a member of game.